<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectoryDlgClassClass</class>
 <widget class="QDialog" name="DirectoryDlgClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DirectoryDlgClass</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblDrives">
       <property name="text">
        <string>lblDrives</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxDrives">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblPath">
       <property name="text">
        <string>lblPath</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtPath">
       <property name="text">
        <string>edtPath</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblDirectories">
     <property name="text">
      <string>lblDirectories</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lbDirectories"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>btnOk</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>btnCancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
